src/bugreporter/odemis_bugreporter.py

```diff
@@ -0,0 +1,542 @@
+        summary_lbl = wx.StaticText(panel, wx.ID_ANY, "Summary:")
+        summary_ctrl = wx.TextCtrl(panel, wx.ID_ANY, size=(500, 23))
+        summary_sizer.Add(summary_lbl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        summary_sizer.Add(summary_ctrl, 10, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 5)
+        sizer.Add(summary_sizer)
+        description_sizer1 = wx.BoxSizer(wx.HORIZONTAL)
+        description_sizer2 = wx.BoxSizer(wx.HORIZONTAL)
+        description_lbl = wx.StaticText(panel, wx.ID_ANY, "Description:")
+        description_ctrl = wx.TextCtrl(panel, wx.ID_ANY, value=DESCRIPTION_DEFAULT_TXT,
+                                       size=(self.GetSize()[0], 400), style=wx.TE_MULTILINE)
+        description_sizer1.Add(description_lbl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        description_sizer2.Add(description_ctrl, 5, wx.EXPAND | wx.ALIGN_LEFT | wx.LEFT | wx.RIGHT, 10)
+        sizer.Add(description_sizer1)
+        sizer.Add(description_sizer2)
+
+        # GDPR text
+        gdpr_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        gdpr_lbl = wx.StaticText(panel, -1, GDPR_TEXT)
+        gdpr_lbl.Wrap(gdpr_lbl.GetSize().width)
+        font = wx.Font(10, wx.NORMAL, wx.ITALIC, wx.NORMAL)
+        gdpr_lbl.SetFont(font)
+        gdpr_sizer.Add(gdpr_lbl, 10, wx.EXPAND | wx.ALIGN_LEFT | wx.ALL, 10)
+        sizer.Add(gdpr_sizer, wx.EXPAND)
+
+        # Cancel and send report buttons
+        button_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        cancel_btn = wx.Button(panel, wx.ID_ANY, "Cancel")
+        cancel_btn.Bind(wx.EVT_BUTTON, self.on_close)
+        button_sizer.Add(cancel_btn, 0, wx.ALL, 10)
+        report_btn = wx.Button(panel, wx.ID_ANY, "Report")
+        report_btn.Bind(wx.EVT_BUTTON, self.on_report_btn)
+        button_sizer.Add(report_btn, 0, wx.ALL, 10)
+
+        # Status update label
+        # TODO: replace by an animated throbber
+        wait_lbl = wx.StaticText(panel, wx.ID_ANY, "")
+        sizer.Add(wait_lbl, 10, wx.EXPAND | wx.ALL, 10)
+        sizer.Add(button_sizer, 0, wx.ALIGN_RIGHT | wx.ALL, 10)
+
+        panel.SetSizer(sizer)
+
+        self.Bind(wx.EVT_CLOSE, self.on_close)
+        self.Centre()
+        self.Show()
+        self.Layout()
+
+        # Make these elements class attributes to easily access the contents
+        self.panel = panel
+        self.name_ctrl = name_ctrl
+        self.email_ctrl = email_ctrl
+        self.summary_ctrl = summary_ctrl
+        self.description_ctrl = description_ctrl
+        self.wait_lbl = wait_lbl
+
+        self.bugreporter = controller
+
+        # flag is set to False when the backspace is pressed
+        self.make_suggestion = True
+
+        # Load known users, if not available make tsv file
+        conf_path = os.path.join(os.path.expanduser(u"~"), '.config', 'odemis', 'bugreporter.tsv')
+        with open(conf_path, 'a+') as f:
+            reader = csv.reader(f, delimiter='\t')
+            self.known_users = OrderedDict(reader)
```

Comment:
Instead of storing the `known_users` dict encoded in utf-8, I'd recommend to store it in unicode (ie, already decoded). This way, you only need to decode once, here, when reading the file, and encode once, when writing the file, in `store_user_info()`. That would avoid all the encode/decode calls whenever a suggestion is done.